apps/web/components/team/MemberListItem.tsx

```diff
@@ -24,6 +24,7 @@ import ConfirmationDialogContent from "@components/dialog/ConfirmationDialogCont
 import Avatar from "@components/ui/Avatar";
 import ModalContainer from "@components/ui/ModalContainer";
 
+import { useMeQuery } from "../Shell";
```

Comment:
Is this is going to be used on another file, consider splitting it into a separate file so we can avoid circular dependency issues.